views/check-your-answers.html

```diff
@@ -48,6 +48,15 @@ <h2 class="govuk-heading-m">
             },
             value: {
               text: objection.created_by.fullName
+            },
+            actions: {
+              items: [
+                {
+                  href: "/objecting-entity-name/?id=1",
```

Comment:
I don't think we need an id? as there will only be one name per application. if id is hardcoded to 1 then it will always be the same.